### DISCUSSION TITLE
Accounts disappeared from the account manager

### DISCUSSION THREAD

**Discussion by ClankJake** [hrfee/jfa-go#69] [General]
I'm getting this error in version 0.3.2
![Screenshot_2021-03-23-12-35-04-889_com server auditor ssh client](https://user-images.githubusercontent.com/44306273/112173981-cfc42d00-8bd4-11eb-86de-9bf86ba3e3ff.jpg)

And the accounts do not appear in the manager
![IMG_20210323_124445](https://user-images.githubusercontent.com/44306273/112174950-95a75b00-8bd5-11eb-862a-aafd3bf636ef.jpg)

**Comment by hrfee** [comment]
Jellyfin seems to be spitting out another time format, thought I'd accounted for them all. fc6b6a9 should fix it.

**Reply by ClankJake** [reply]
The error still persists. It does not load the bills. As he had an old profile there he normally creates on jellyfin but he doesn't even load this new user.

**Reply by hrfee** [reply]
Is there the same error in the logs? Testing with the time I saw in your logs worked.

**Reply by ClankJake** [reply]
Yes the same error appears in the log.

**Reply by hrfee** [reply]
This is a weird bug, I've just looked at my server logs and it happened to me too but stopped after a restart. I've rewritten the responsible function in 9875458, so now it should cover pretty much any variation of the time format. Can you update and try it?

**Reply by ClankJake** [reply]
show!! I updated here now the accounts are back up.
